docs/webapp/applications/apps_overview.md

```diff
@@ -13,10 +13,6 @@ Use ClearML's GUI Applications to manage ML workloads and automatically run your
 Configure and launch app instances, then track their execution from the app dashboard.
 
 ClearML provides the following applications:
-* [**GPU Compute**](apps_gpu_compute.md) - Launch cloud machines on demand and optimize their usage according to a 
```

Comment:
@pollfly Put in a note on autoscalers used to be here and are now in orchestration (linked)